src/uct/api/uct.h

```diff
@@ -2626,6 +2630,40 @@ UCT_INLINE_API ucs_status_t uct_ep_am_short(uct_ep_h ep, uint8_t id, uint64_t he
 }
 
 
+/**
+ * @ingroup UCT_AM
+ * @brief Short send operation.
+ *
+ * This routine sends a message using @ref uct_short_protocol_desc "short" protocol.
+ * The input data in @a iov array of @ref ::uct_iov_t structures is sent to remote
+ * side ("gather output"). Buffers in @a iov are processed in array order.
+ * This means that the function completes iov[0] before proceeding to
+ * iov[1], and so on.
+ *
+ * @param [in] ep              Destination endpoint handle.
+ * @param [in] id              Active message id. Must be in range 0..UCT_AM_ID_MAX-1.
+ * @param [in] header          Active message header.
+ * @param [in] iov             Points to an array of @ref ::uct_iov_t structures.
+ *                             The @a iov pointer must be a valid address of an array
+ *                             of @ref ::uct_iov_t structures. A particular structure
+ *                             pointer must be a valid address. A NULL terminated
+ *                             array is not required.
```

Comment:
shall we comment that stride/count fields in uct_iov_t are ignored?
or maybe pass linux's `struct iovec`

Reply:
Ok, I will comment that stride and count fields are ignored.